### DISCUSSION TITLE
Exploring possibility of moving WP install/Meeting Guide to new folder

### DISCUSSION THREAD

**Discussion by ginageick** [code4recovery/12-step-meeting-list#1331] [Q&A] [answered]
Hi there... 
I'm thinking of moving wp installation/meeting guide to a different folder (original was unfortunate/poor naming)... what will that do (if anything) to the daily feed? Is there anything to be done ahead of time?

Thanks so much,
Gina, for www.area84aa.org

**Comment by kiyote33** [comment]
Hi @ginageick,
 Just to clarify your question, I am assuming that you are considering moving your WordPress site to a new folder under your hosting provider which typically places your Area 84 website now under something with a path like "/home1/your_userid/public_html/WP install/Meeting Guide"? 
 If the above is true, you may need to change your DNS records when changing. The simplest solution may be to work with your hosting provider to move the site and get the DNS configured correctly. Hope this helps!

**Comment by ginageick** [comment]
No, our DNS is fine. Wordpress ONLY handles the meeting guide. I detest Wordpress LOL 
I want to move that part of the website into a folder off of the root. But my question is whether a different path affects the feed for the meeting guide?

**Comment by kiyote33** [comment]
Ah, thanks for the clarification Gina. This is the support site for TSML - I can't say for sure what the pathing requirements are for the Meeting Guide app. Please contact GSO at [appsupport@aa.org](mailto:appsupport@aa.org) to discuss your issue.

You can get  technical info here: [https://meetingguide.helpdocs.io/category/eebenj2xql-connecting-to-meeting-guide](https://meetingguide.helpdocs.io/category/eebenj2xql-connecting-to-meeting-guide). Hope this helps...

**Comment by SIA-WEBSUP** [comment]
Gina,
I'm fairly certain that If you are moving your WP install from your root to a subfolder off of the root and you do nothing else, then yes the feed will also have moved 
    from: 'https://your-site.org/wp-admin/admin-ajax.php?action=meetings' 
    to:     'https://your-site.org/[WP install folder]/wp-admin/admin-ajax.php?action=meetings'

If you can live with this longer path to get to the WP install, including your /meetings page, then you would need to let the meeting guide app know the new path to your feed.

Please let me know if I misunderstood your question.

**Comment by SIA-WEBSUP** [comment, marked as answer]
BTW, here's the link to your feed currently:
    https://area84aa.org/wp/wordpress/wp-admin/admin-ajax.php?action=meetings
    
Not sure if you've already moved your wordpress install to where you'd like it to be.

**Comment by ginageick** [comment]
Awesome, thanks so much, guys. And thank you for pointing out the actual link to the feed! 
I figured that a different path would impact the feed... I just wanted to confirm that before I went and did anything stupid :)

Much appreciated!
Gina G.
